libcontainer/container.go

```diff
@@ -60,8 +60,8 @@ type State struct {
 //
 // Each container is thread-safe within the same process. Since a container can
 // be destroyed by a separate process, any function may return that the container
-// was not found.
-type Container interface {
+// was not found. CommonContainer includes methods that are platform agnostic.
+type CommonContainer interface {
```

Comment:
Totally fair comment. I used common as I'd used the same type of refactoring multiple times in docker codebase. Certainly open to suggestions/alternatives :)